### PR TITLE
Consolidate Dockerfiles

### PR DESCRIPTION
**Pull request by djbingham**
https://eaflood.atlassian.net/browse/PSD-519

- Correct image repository names
- Remove `ENTRYPOINT` from parent images so shell-form commands can be used
- Combine Dockerfiles using multi-stage builds, similar to Node parent image repository

### CODE REVIEW

**Comment by johnwatson484** [comment]
The image names in the README also need updating to reflect these changes

**Comment by johnwatson484** [comment]
Slightly nitpicky as not sure if in scope of this PR, but the year in the licence should be 2020

**Review by johnwatson484 — COMMENTED** [review]
Dockerfiles look good, just needs some minor text changes.

**Comment by djbingham** [comment]
Thanks @johnwatson484 

I have:

1. updated the image names in the README
2. removed comments from the Dockerfiles on setting the `ASPNETCORE_ENVIRONMENT` variable - it seems self-evident what that's doing
3. updated the license year to 2020

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ This repository contains .Net Core parent Docker image source code for the Futur
 
 Two parent images are created from this repository:
 
-- `ffc-dotnetcore-runtime`
-- `ffc-dotnetcore-sdk`
+- `ffc-dotnetcore`
+- `ffc-dotnetcore-develpment`
```
Comment (johnwatson484) [inline README.md]: Slight spelling mistake

Comment (djbingham) [reply README.md]: Thanks. Corrected.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**